### ISSUE TITLE
Support clojure.core/tap>

### ISSUE THREAD

**Issue by devurandom** [afucher/clojure-repl-intellij#79]
**Describe the bug**

It would be nice if values captures using `clojure.core/tap>` would be shown somewhere in the IDE (ideally next to the code that emitted it), both when evaluating something on the REPL and also when running tests.

----

Right now I cannot even evaluate `(tap> {:test 1})`:
```
;; Connected to nREPL server - nrepl://localhost:7888
;; Clojure REPL Intellij 1.3.1
;; Clojure 1.11.3, Java 11.0.22, nREPL 1.1.1

user> {:test 1})

             java.lang.RuntimeException: Unmatched delimiter: )
clojure.lang.LispReader$ReaderException: java.lang.RuntimeException: Unmatched delimiter: )
             clojure.lang.ExceptionInfo:

user> 

             java.lang.RuntimeException: Unmatched delimiter: )
clojure.lang.LispReader$ReaderException: java.lang.RuntimeException: Unmatched delimiter: )
             clojure.lang.ExceptionInfo:


;; => {:test 1}
user> {:test 1})

             java.lang.RuntimeException: Unmatched delimiter: )
clojure.lang.LispReader$ReaderException: java.lang.RuntimeException: Unmatched delimiter: )
             clojure.lang.ExceptionInfo:

user> 

             java.lang.RuntimeException: Unmatched delimiter: )
clojure.lang.LispReader$ReaderException: java.lang.RuntimeException: Unmatched delimiter: )
             clojure.lang.ExceptionInfo:


;; => {:test 1}
user> {:test 1})

             java.lang.RuntimeException: Unmatched delimiter: )
clojure.lang.LispReader$ReaderException: java.lang.RuntimeException: Unmatched delimiter: )
             clojure.lang.ExceptionInfo:

user> 

             java.lang.RuntimeException: Unmatched delimiter: )
clojure.lang.LispReader$ReaderException: java.lang.RuntimeException: Unmatched delimiter: )
             clojure.lang.ExceptionInfo:


;; => {:test 1}
user> 
```

Each time I entered `(tap> {:test 1})`, but the Clojure REPL plugin seems to only pass ` {:test 1})` to the REPL.

**Comment by isaiasvallejos** [comment]
@devurandom I understand that the suggestion isn't equal to that, but for the sake of knowledge, I use the [Portal](https://github.com/djblue/portal) plugin for IntelliJ. It works well with `tap>` and achieves a similar effect to what you suggested (print all `tap>` in a tab and you can interact with the data in a visual matter).

Maybe can be used as a inspiration also.

**Comment by ericdallo** [comment]
Ah that's a bug because of the way we parse the `>` character in the repl, I'll take a look

**Comment by ericdallo** [comment]
Fixed in 1.4.1! Thanks @lread for the help!

**Comment by devurandom** [comment]
Just out of curiousity, after skimming the tests added in https://github.com/afucher/clojure-repl-intellij/commit/74c904300a3df19dcccc5ef2283596cb33d28c3b: Why do you have to parse several request/response pairs and extract the last? Is that a restriction of nREPL, that it does not deal with structured data?
